proto/ibc/core/port/v1/query.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package ibc.core.port.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/core/05-port/types";
+
+import "ibc/core/channel/v1/channel.proto";
+
+// Query provides defines the gRPC querier service
+service Query {
+  // Port queries an IBC Port.
+  rpc Port(QueryPortRequest) returns (QueryPortResponse) {}
+}
+
+// QueryPortRequest is the request type for the Query/Port RPC method
+message QueryPortRequest {
+  // port unique identifier
+  string port_id = 1;
+  // counterparty channel end
+  ibc.core.channel.v1.Counterparty counterparty = 2;
+  // counterparty version
+  string counterparty_version = 3;
+}
+
+// QueryPortResponse is the response type for the Query/Port RPC method.
+message QueryPortResponse {
+  // port id associated with the request identifiers
+  string port_id = 1;
```

Comment:
Do we need this?

Reply:
No, it's not required as the caller will already have this, but semantically it may be nice to return a complete response where it's explicit that the following port id and version are coupled. 

I don't feel particularly strongly about this though, I can remove if needs be!

Reply:
I don't feel particularly strong about this either. @seantking thoughts?

Reply:
I say leave it in, sounds good. :+1: